nuxt.config.js

```diff
@@ -1,6 +1,9 @@
 import theme from '@nuxt/content-theme-docs'
 
 export default theme({
+  router: {
+    trailingSlash: true,
+  },
```

Comment:
This change appears to make the Netlify deploy preview work, we may also add the following to the config:

```
   sitemap: {
     hostname: 'https://docs.sigstore.dev',
     trailingSlash: true,
   },
```